src/Xtools/Repository.php

```diff
@@ -134,6 +138,29 @@ public function isLabs()
     }
 
     /**
+     * Make a request to the XTools API. This requires that app.multithread.api_url be set.
+     * @param string $endpoint Relative path to endpoint with relevant query parameters.
+     * @param bool $async Set to true to asynchronously query and return a promise.
+     * @return mixed|GuzzleHttp\Promise\Promise
+     */
+    public function queryXTools($endpoint, $async = false)
```

Comment:
I got confused here (not hard to do, certainly). Should this be called `queryXToolsApi()`?